index.bs

```diff
@@ -2743,6 +2743,27 @@ and makes it clear when the state described by the flags is reset.
 
 <!-- TODO: add examples -->
 
+<h3 id="monkey-patching">Avoid monkey patching</h3>
+A <dfn export>monkey patch</dfn> layers new functionality on top of an existing specification in a way that extends, overrides, or otherwise modifies the underlying specification's behavior.
```

Comment:
```suggestion
A <dfn export>monkey patch</dfn> layers new functionality on top of an existing specification in a way that extends, overrides, or otherwise modifies the existing specification's behavior.
```